wrappers/bowtie2/build/wrapper.py

```diff
@@ -0,0 +1,15 @@
+__author__ = "Daniel Standage"
+__copyright__ = "Copyright 2020, Daniel Standage"
+__email__ = "daniel.standage@nbacc.dhs.gov"
+__license__ = "MIT"
+
+
+from snakemake.shell import shell
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+indexbase = snakemake.output[0].replace(".1.bt2", "")
+shell(
+    "bowtie2-build --threads {snakemake.threads} {snakemake.params.extra} "
+    "{snakemake.input.reference} {indexbase}"
+)
```

Comment:
Tu set des logs mais tu ne l'utilises pas 😅

Reply:
Si l'idée de 'extra' c'est toutes les options en plus, on va suivre la converntion actuelle: 'options' a la place de extra
Par exemple tous les fichiers de config aujourd'hui on cette stucture:

    rule_name:
        specific_param1: xxx
        specific_param2: xxx
        options: "toules les parameters et options possible et imaginable


A changer dans tous les wrappers. 
En fait, je pense qu'il faut qu'on fasse cela pour tous les wrappers qui execute un outil standard style bowtie, fastqc, etc

C'est a a dire qu'on doit avoir :+1: 

    options = snakemake.params.get("options", "")

puis si par exemple shell command:

    shell("bowtie ....... {extra}

D'ailleurs ici le code si on utulise snakemake.params, alors la ligne 9 est pas utilisée

Reply:
C'est noté et corrigé !